lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -338,6 +338,19 @@ def write(chunk)
       # Group the entries since we have to make one call per tag.
       grouped_entries = {}
       chunk.msgpack_each do |tag, *arr|
+        tag = tag.to_s if tag.is_a?(Integer)
```

Comment:
I was debating how strict we should treat the tags. `require_valid_tags` is a good idea, so we can treat it differently based on user's preference. Just changed the implementation to be customizable.